test/google_crawler_web/controllers/api/upload_controller_test.exs

```diff
@@ -0,0 +1,65 @@
+defmodule GoogleCrawlerWeb.Api.UploadControllerTest do
+  use GoogleCrawlerWeb.ConnCase, async: true
+  alias GoogleCrawler.Repo
+  alias GoogleCrawler.Search.Keyword
+
+  describe "POST create/2" do
+    test "creates the keywords when file is succesfully processed", %{conn: conn} do
```

Comment:
```suggestion
    test "creates the keywords when file is successfully processed", %{conn: conn} do
```
There are also the same typos below 📝

Reply:
Fixed in https://github.com/ankitkalia1195/elixir-google-crawler/pull/54/commits/48ec8848c699b401bb780eb0134a25aeeed09f8a